openlibrary/core/sponsorships.py

```diff
@@ -1,19 +1,84 @@
+import json
+import urllib
 import requests
 from infogami import config
+from openlibrary.accounts import get_internet_archive_id
+from openlibrary.core.lending import config_ia_civicrm_api
 
-CIVI_SPONSOR_API = '%s/internal/fake/civicrm' % config.get('servername', 'http://localhost')
+CIVI_ISBN = 'custom_52'
+CIVI_USERNAME = 'custom_51'
+CIVI_CONTEXT = 'custom_53'
 
-def get_all_sponsors():
-    """
-    Query civi for a list of all sponsorships, for all users.
-    These results will have to be summed by user for the
-    leader-board and then cached
-    """
-    pass  # todo, later
 
-def get_sponsored_editions(archive_username, limit=50, offset=0):
+def get_sponsored_editions(user, limit=50, offset=0):
```

Comment:
@mekarpeles can we remove `limit` and `offset` from function parameter list as they are not being used anywhere in the function